fluentd/wait_for_es_version.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Given you already have ruby available, consider writing the entire script as ruby instead of shuffling back and forth to bash https://ruby-doc.org/stdlib-2.4.1/libdoc/net/http/rdoc/Net/HTTP.html

Reply:
my thought was it was easy to do the curl in bash vs implementing configuring and using a http client in ruby